pyproject.toml

```diff
@@ -44,6 +45,8 @@ test = [
     "pytest-cov",
     "psycopg2-binary",
     "mysql-connector-python",
+]
```

Comment:
```suggestion
    "mysql-connector-python",
    "sqlacodegen[sqlmodel]",
]
```

Reply:
This will cause the sqlmodel tests to actually run, and restore the lost coverage.

Reply:
@agronholm [ sqlmodel is not compatible with sqlalchemy 2.0](https://github.com/tiangolo/sqlmodel/blob/main/pyproject.toml#L34), this makes it tricky.

Reply:
Oh right. Hmm, those tests need to be run _somehow_.

Reply:
I think you had the right idea, but couldn't you just have compared the sqlalchemy version to the string `1.4.*` and conditionally installed sqlmodel then?

Reply:
There are many roads that lead to Rome. This way has the advantage that if one wants to pin SQLalchemy to a certain version only the install line needs to be changed.

Reply:
I'm not going to want to pin it to a specific version.

Reply:
This seems resolved to me since https://github.com/agronholm/sqlacodegen/pull/249/commits/bbfacf7eca7b7a13309a127d672fe885af8686a8, see lines 24-29